src/test/java/testOracle/CombatTest.java

```diff
@@ -0,0 +1,521 @@
+package testOracle;
```

Comment:
Is there are reason why package is named testOracle? This could be also under org.openRealmOfStars.player.combat
just rename the file CombatTest2.java